### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.6 - es. 3.66 (23 ottobre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#301] [Esercizi M-Z]
<img width="965" alt="immagine" src="https://github.com/user-attachments/assets/0f2f796b-d7ad-4530-8e33-af07c2c8127d">

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/531dd8c4-c8fd-4d0b-ac33-8586dc8442db)
![immagine](https://github.com/user-attachments/assets/dbeeb11c-5c24-40fd-a2bc-1ad70eedae95)
